stake-pool/py/pyproject.toml

```diff
@@ -0,0 +1,67 @@
+[project]
+name = 'stake-pool'
+version = '1.0'
+authors = [
+  { name= 'joncinque'},
+]
+description = 'solana labs stake pool reference implementation'
+readme = 'README.md'
+license = { file='LICENSE' }
+requires-python = '>=3.7'
+classifiers = [
+    'Programming Language :: Python :: 3',
+    'License :: OSI Approved :: Apache Software License',
+    'Operating System :: OS Independent',
+]
+dependencies = [
+        'anyio==3.3.4',
+        'attrs==21.2.0',
+        'base58==2.1.0',
+        'cachetools==4.2.4',
+        'certifi==2021.10.8',
+        'cffi==1.15.0',
+        'charset-normalizer==2.0.7',
+        'construct==2.10.67',
+        'flake8==4.0.1',
+        'h11==0.12.0',
+        'httpcore==0.13.7',
+        'httpx==0.20.0',
+        'idna==3.3',
+        'iniconfig==1.1.1',
+        'mccabe==0.6.1',
+        'mypy==0.910',
+        'mypy-extensions==0.4.3',
+        'packaging==21.2',
+        'pluggy==1.0.0',
+        'py==1.10.0',
+        'pycodestyle==2.8.0',
+        'pycparser==2.20',
+        'pyflakes==2.4.0',
+        'PyNaCl==1.4.0',
+        'pyparsing==2.4.7',
+        'pytest==6.2.5',
+        'pytest-asyncio==0.16.0',
+        'requests==2.26.0',
+        'rfc3986==1.5.0',
+        'six==1.16.0',
+        'sniffio==1.2.0',
+        'solana==0.18.1',
+        'toml==0.10.2',
+        'typing-extensions==3.10.0.2',
+        'urllib3==1.26.7'
+      ]
+
+[tool.setuptools]
+packages = ['bot', 'vote', 'stake', 'system', 'spl_token', 'stake_pool']
+
+[project.urls]
+'Homepage' = 'https://github.com/solana-labs/solana-program-library/new/master/stake-pool/py'
```

Comment:
This page actually creates a new file in the repo, so let's do https://github.com/solana-labs/solana-program-library/tree/master/stake-pool/py instead